docker-compose.override.yml

```diff
@@ -103,12 +103,20 @@ services:
     #restart: always
     ports:
      - 8074:80
-    networks: 
+    networks:
      - internal-net
      - external-net
     volumes:
      - /sessions
 
+  #########################################################
+  ## End-to-end tests #####################################
+  #########################################################
+  e2e-tests:
+    image: gradido/e2e-tests

```

Comment:
This looks a bit dangerous, since we do not have this image on dockerhub currently.

Reply:
Yes, this is not necessary. Will be removed.